app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,7 @@
         android:roundIcon="@drawable/app_icon_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".activity.NewCompassActivity"></activity>
```

Comment:
Why is this renamed to "new"? Can't we have the same class with changes?

Reply:
I tried using the same name. There were some unexpected issues with the PSLab Sensor model. That's why changed the name. There some issues with ubinder. It can't find the 'compass' view when the activity name is kept the same. I could not figure out the reason. Can you look once by yourself

Reply:
@neel1998 Im sure changing the name never give unexpected issues.

Reply:
Probably there were caches and files were dirty. Have you tried a clean rebuild? :thinking:

Reply:
I rebuilt. It is working fine now. Sorry for my mistake.